a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -814,17 +814,15 @@ <h3 data-localization-id="navigation-title">Navigation</h3>
 				<div class="note">
 					<p>This key information can be hidden if metadata is missing. Alternatively it can be stated that
 							<span data-localization-id="navigation-no-metadata" data-localization-mode="descriptive">No
-							information about navigation is available.</span>
-					</p>
+							information about navigation is available</span>.</p>
 				</div>
 
 				<p>Identifies the navigation features included in the publication.</p>
 
-				<p>Offering different ways to navigate digital publications (table of contents, by headings, index,
-					etc.) is important to allow the user to easily access each part of the content. These features are
-					both essential for accessibility and very important for all users reading publications. Therefore,
+				<p>   Navigating a digital publication    through a table of contents, an index, by headings,  or by direct go to page shortcuts is important to allow the user to easily access each part of the digital publication. These features are
+					both essential for accessibility and  important for all users reading publications. Therefore,
```

Comment:
The "proper title selection" phrasing sounds awkward. Could this be written instead as: "Therefore, it is important to clearly communicate what navigation features are available so users can decide if a publication will meet their needs."